docs/install.md

```diff
@@ -20,19 +37,12 @@ brew install brimdata/tap/super
 
 Once installed, run a [quick test](#quick-tests).
 
-## Binary Download
-
-We offer pre-built binaries for macOS, Windows and Linux for both amd64/x86 and arm
-architectures in the super [GitHub Release page](https://github.com/brimdata/super/releases).
-
-Once the `super` binary is unpacked from a downloaded package, run a [quick test](#quick-tests).
-
 ## Building From Source
 
 If you have Go installed, you can easily build `super` from source:
 
 ```bash
-go install github.com/brimdata/super/cmd/super@latest
+go install github.com/brimdata/super/cmd/super@main
```

Comment:
`@latest` would still get them the last GA Zed v1.18.0 release, which is not what we want. Since the README/docs changes in this PR set expectations that they're installing code that's still under development, tip-of-`main` is exactly what we want them to do for now.